generators/generator-microsoft-bot-empty/generators/app/templates/recognizers/emptyBot.en-us.lu.dialog

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
So with the Empty bot template, I think the idea was that we want to specifically avoid having any recognizers configured as the intent is it should just echo back what is said to it. @ryanlengel can you advise?

Reply:
Without this file a generated JS bot does not start, the runtime throws an exception due to a file not being found. I think we want all the generators to produce a minimal (but working) bot.

Reply:
Agreed 100%. I guess I just want to make sure that it's expected we need these files, or if it's somehow a regression in Composer. It could be that we just missed them the first time around, as I would expect that what was there originally should have been done with the intent it work out of the box.

Reply:
If I recall we didn't include in the recognizers but saw that Composer was adding them when generating a template anyways. I thought that we had an issue tracking this (regarding tracking down what files the generators should provide vs what Composer was doing), but I can't find it in here (maybe it was in the core repo?). We definitely need these files to work, when iterating on existing generators these are produced every time and we're currently deleting them before making commits.

The only thing I'm unsure of is if any of the names here have to convert to the botName variable that is provided?

Reply:
The runtime code uses the keys present in the main dialog to locate files. They _could_ be named after the bot but I don't think it's strictly required.